go/client/chat_api_listen_display.go

```diff
@@ -37,22 +36,12 @@ func newMsgNotification(source string) *chat1.MsgNotification {
 }
 
 func (d *chatNotificationDisplay) setupFilters(ctx context.Context, channelFilters []ChatChannel) error {
-	d.filtersNormalized = channelFilters
-	for i, v := range channelFilters {
-		v.MembersType = strings.ToUpper(v.MembersType)
-		v.TopicType = strings.ToUpper(v.TopicType)
-		if v.MembersType != "TEAM" {
-			// We are looking in inbox for conversations between users
-			// to normalize display name without using resolve RPC.
-			conv, _, err := d.svc.findConversation(ctx, "", v)
-			if err != nil {
-				return fmt.Errorf("Unable to find chat channel for %q: %s", v.Name, err.Error())
-			}
-			v.Name = conv.Info.TlfName
-		} else {
-			v.Name = strings.ToLower(v.Name)
+	for _, v := range channelFilters {
+		conv, _, err := d.svc.findConversation(ctx, "", v)
```

Comment:
i thought it would have to find all conversations in the team somehow to be able to support the "wildcard" for "listen to all conversations im in for the team" from line 138

Reply:
Is anything using that functionality right now?

Reply:
Hard to tell. Would the same arguments start only following `#general` now with this change?

Reply:
I fixed it.